detox/test/ios/example.xcodeproj/project.pbxproj

```diff
@@ -491,10 +519,14 @@
 				39A34C5D1E30ED3600BEBB59 /* libcxxreact.a */,
 				39A34C5F1E30ED3600BEBB59 /* libjschelpers.a */,
 				39A34C611E30ED3600BEBB59 /* libjschelpers.a */,
+				D58447B0204E9C6000248543 /* libjsinspector.a */,
```

Comment:
What is this? What will happen if old RN is used? In CI, we have an RN matrix that changes RN versions on the fly. Currently we are testing RN 51 and 53.3